internal/xds/translator/testdata/in/xds-ir/listener-connection-limit.yaml

```diff
@@ -44,26 +44,28 @@ tcp:
   connection:
     limit:
       value: 3
-  tls:
-    passthrough:
-      snis:
-      - bar.com
-  destination:
-    name: "tls-route-dest"
-    settings:
-    - endpoints:
-      - host: "1.2.3.4"
-        port: 50000
+  routes:
+  - tls:
+      inspector:
+        snis:
+        - bar.com
+    destination:
+      name: "tls-route-dest"
+      settings:
+      - endpoints:
+        - host: "1.2.3.4"
+          port: 50000
 - name: "fourth-listener"
   address: "0.0.0.0"
   connection:
     limit:
       value: 10
       closeDelay: 3s
   port: 10083
-  destination:
-    name: "tcp-route-dest"
-    settings:
-    - endpoints:
-      - host: "1.2.3.4"
-        port: 50000
+  routes:
```

Comment:
This is a bug in the listener-connection-limit test. The TCPRoute in the TCP Listener ir is incorrect.